Update and rename study-sites.md to location-ids.md

goddesswarship opened the pull request:
Removed stray punctuation from Docwriter migration 
Replaced "study sites" with "location ID"
Formatted JSON code
Resized screenshots
Replaced link to locationID.json in repository with a downloadable file.

goddesswarship committed:
Update and rename study-sites.md to location-ids.md

Removed stray punctuation from Docwriter migration
Replaced "study sites" with "location ID"
Formatted JSON code

goddesswarship committed:
Delete docs/assets/images/wb-encounter-locations.png

Remove image to replace with smaller version of the same image

goddesswarship committed:
Add files via upload

Resized image

goddesswarship committed:
Delete docs/assets/images/wb-submit-locationid.png

Remove image to replace with smaller version of the same image

goddesswarship committed:
Add files via upload

Resized image

goddesswarship committed:
Delete docs/assets/images/wb-encounter-customalg.png

Remove image to replace with smaller version of the same image

goddesswarship committed:
Add files via upload

Resized image

goddesswarship committed:
Add files via upload

Adding Location ID JSON template

goddesswarship committed:
Update location-ids.md

TanyaStere42 commented on docs/assets/templates/locationID_template.json:
```diff
@@ -0,0 +1,114 @@
+{
+	"description": "Hierarchical Encounter.locationID list in Wildbook",
+	"source": "http://en.wikipedia.org/wiki/List_of_seas",
```
not gonna stop the PR for this, but we gotta remove "source" or at least make it accurate. This is not a List of seas from wikipedia 😆

goddesswarship replied:
Fixed!

TanyaStere42 commented on docs/data/location-ids.md:
```diff
@@ -16,22 +16,46 @@ When you [report an Encounter](report-encounter.md), you can set its **location
+## Adding a Location ID
 
-To add a new study site, you'll need to contact your Wildbook's administrator or make the request at the [Wildbook Community](https://community.wildme.org).
+To add a new Location ID, you'll need to contact your Wildbook's administrator or make the request at the [Wildbook Community](https://community.wildme.org).
```
should we add that they can make a PR against their platform branch? or handle that in a different page?

goddesswarship replied:
Good call! I've added a link to our PR page as a third option.

TanyaStere42 reviewed (approved):
Comments aren't blocking; just points to think on. I'm gonna approve now and you can either revise or merge immediately

goddesswarship committed:
Delete docs/assets/templates/locationID_template.json

Deleting to replace with updated template.

goddesswarship committed:
Add files via upload

Removed source line with wikipedia link

goddesswarship committed:
Update location-ids.md

Updated instructions for adding a new location ID so that it includes creating a PR.
